src/algebra/symmetrized.lean

```diff
@@ -210,4 +210,38 @@ lemma mul_comm [has_mul α] [add_comm_semigroup α] [has_one α] [invertible (2
   a * b = b * a :=
 by rw [mul_def, mul_def, add_comm]
 
+/-- `2` commutes with every element of a ring. -/
+lemma two_commute {α : Type*} [ring α] (a : α) : commute 2 a := (commute.one_left a).bit0_left
+
+/- If 2 is invertible, ⅟2 commutes with every element of a ring -/
+lemma half_commute {α : Type*} [ring α] [invertible (2 : α)] (a : α) : commute (⅟2) a :=
+(two_commute a).inv_of_left
```

Comment:
Could you either
* inline those lemmas
* write a tactic to generate those lemmas
* move them to an earlier file (and write a todo for the tactic that would generate them)

Reply:
I've inlined them, thanks.

Reply:
I guess the tactic that @YaelDillies is suggesting is something to produce lemmas about numerals commuting? Eitherway, out of scope here.